### DISCUSSION TITLE
第1节课笔记(4班)

### DISCUSSION THREAD

**Discussion by shiruo2686** [InternLM/Tutorial#47] [笔记]
4班的笔记请提交到这里

**Comment by CastleDream** [comment]
https://stitch.blog.csdn.net/article/details/135372922

**Reply by FlamingoPg** [reply]
同学，贴一下你的群昵称呀。方便给你打分

**Comment by shiruo2686** [comment]
释若的第1次笔记：https://blog.csdn.net/weixin_60280805/article/details/135374661

**Comment by suxinghang** [comment]
https://blog.csdn.net/qq_36046307/article/details/135387892?spm=1001.2014.3001.5501

**Reply by FlamingoPg** [reply]
同学，贴一下你的群昵称呀。方便给你打分

**Comment by helonggood** [comment]
【书生·浦语大模型全链路开源体系 - CSDN App】http://t.csdnimg.cn/mLAdC

**Comment by billfjj** [comment]
肆拾贰的第一节课笔记https://blog.csdn.net/baidu_39417409/article/details/135397726?spm=1001.2014.3001.5502

**Comment by zhjunqin** [comment]
第一节课笔记 https://blog.csdn.net/zhjunqin/article/details/135397770?spm=1001.2014.3001.5502

**Comment by canglangzhishuiz** [comment]
第一节课笔记：http://t.csdnimg.cn/jXJIB

**Comment by onlyzer0** [comment]
第一节课笔记：https://blog.csdn.net/qq_42907812/article/details/135429280?spm=1001.2014.3001.5501

**Comment by lwttttt** [comment]
第一节课笔记 ：https://blog.csdn.net/m0_53408051/article/details/135462628?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135462628%22%2C%22source%22%3A%22m0_53408051%22%7D
群昵称：小土.

**Comment by phbst** [comment]
https://thankful-archeology-738.notion.site/184b3df006f745b583b68750f1d633d5?pvs=4
群：彭宏博
